receiver/splunkenterprisereceiver/README.md

```diff
@@ -20,6 +20,8 @@ jobs.
 
 ## Configuration
 
+By default the Splunk Enterprise receiver is not configured to gather any metrics (i.e. there are no metrics enabled by default).
```

Comment:
Maybe bold this? This is going to get asked about because reading is hard :(

Reply:
i made it bold!